.pylintrc

```diff
@@ -1,265 +1,313 @@
 [MASTER]
 
-# Specify a configuration file.
```

Comment:
Regenerated this file via `pylint --generate-rcfile` (using the most recent pylint version).

Subsequent modifications
- disabled rules that we currently do not fully adhere to (marked with TODO or another comment)
- reintroduced docparams and docstyle plugins we used before